geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStore.scala

```diff
@@ -194,39 +240,24 @@ class AccumuloDataStore(val connector: Connector, override val config: AccumuloD
   }
 
   override protected def onSchemaUpdated(sft: SimpleFeatureType, previous: SimpleFeatureType): Unit = {
-    import org.locationtech.geomesa.utils.geotools.RichAttributeDescriptors.RichAttributeDescriptor
-    // check for newly indexed attributes and re-configure the splits
-    val previousAttrIndices = previous.getAttributeDescriptors.asScala.collect {
-      case d if d.isIndexed => d.getLocalName
-    }
-    if (sft.getAttributeDescriptors.asScala.exists(d => d.isIndexed && !previousAttrIndices.contains(d.getLocalName))) {
-      val partitioned = TablePartition.partitioned(sft)
-      manager.indices(sft).foreach {
-        case s: AttributeSplittable if !partitioned => s.configureSplits(sft, this, None)
-        case s: AttributeSplittable => s.getPartitions(sft, this).foreach(p => s.configureSplits(sft, this, Some(p)))
-        case _ => // no-op
-      }
-    }
+    super.onSchemaUpdated(sft, previous)
     // configure the stats combining iterator on the table for this sft
     stats.configureStatCombiner(connector, sft)
   }
 
-  override def getQueryPlan(query: Query,
-                            index: Option[AccumuloFeatureIndexType],
-                            explainer: Explainer): Seq[AccumuloQueryPlan] =
+  override def getQueryPlan(query: Query, index: Option[String], explainer: Explainer): Seq[AccumuloQueryPlan] =
```

Comment:
Since this is just calling to super, can the call be removed?  

Maybe this is more clear to read?

Reply:
the super method returns the generic `QueryPlan[AccumuloDataStore]`, this specifies that it is an `AccumuloQueryPlan`, which exposes all the bits of the plan like ranges, etc.